src/pages/Task.tsx

```diff
@@ -130,6 +126,9 @@ const TaskCore: React.FC = () => {
 
   usePageTitle(`Task${displayName ? ` - ${displayName}` : ""}`);
 
+  const [failedTask, setFailedTask] = useState(undefined);
+  const [showBuildBaronTab, setshowbuildbarontab] = useState(!failedTask);
```

Comment:
I think we originally included `showBuildBaronTab` to conditionally render all of the leafygreen tabs at the same time. This helped us avoid a bug in leafygreen where we were unable to remove or add tabs under the Tab provider after the Tab provider's initial render. 
I think leafygreen patched that bug and we no longer need `showBuildBaronTab`.

Reply:
Yup i think this can be removed